cloudformation/ag_cloud_ray_aws.yaml

```diff
@@ -26,6 +26,7 @@ Resources:
           - Effect: Allow
             Principal:
               Service: ec2.amazonaws.com
+              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
```

Comment:
@suzhoum FYI, this principal is provided in the `AssumeRolePolicyDocument` for the SageMaker role but not the Ray AWS role. For this reason, it's impossible to assume the `AGCloudRayExecutionRole` outside of an EC2 instance. I have added this permission to the Ray policy to allow users to assume the role.